docs: quick start docs of GreptimeCloud (#510)
GreptimeTeam/docs: docs/greptimecloud/getting-started/quick-start/java.md

> Use the following command collects JVM runtime metrics, such as CPU and memory usage, and sends it to GreptimeCloud. Once the metrics are sent successfully, they can be viewed on the GreptimeCloud dashboard.

Reviewer: ```suggestion
Use the following command to collect JVM runtime metrics, such as CPU and memory usage, and send them to GreptimeCloud. Once the metrics have been sent successfully, they can be viewed on the GreptimeCloud dashboard.
```
